docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   graphelier-app:
     build:
       context: ./app
-      target: dev
+      target: prod
```

Comment:
So from my understanding we'll need to remove `docker-compose.override.yml` from the prod env to activate prod ?

Reply:
no so if you just docker-compose without passing in a file it'll combine both docker-compose.yml and docker-compose.override.yml files. However, if we specify the file, its contents will not get merged with the override file. This allows us to run a dev environment with `docker-compose build && docker-compose up` and a prod environment with `docker-compose -f docker-compose.yml build && docker-compose -f docker-compose.yml up`

Reply:
ok